tests/glm/test_link.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import pytest
 
-from glum._link import Link, LogitLink, LogLink, TweedieLink
+from glum._link import CloglogLink, Link, LogitLink, LogLink, TweedieLink
```

Comment:
For the tests, can you add a comparison with statsmodels: https://www.statsmodels.org/dev/generated/statsmodels.genmod.families.links.CLogLog.html

Reply:
Okay. I can also test it against `glmnet` like [this](https://github.com/Quantco/glum/blob/bfc41d30f6fa4798d669aebac2b08350cc052a3a/tests/glm/test_glm.py#L1011-L1012) if we want to compare the regularized version, too.